src/CommunityToolkit.Maui/Behaviors/Validators/MultiValidationBehavior.shared.cs

```diff
@@ -75,31 +75,27 @@ protected override async ValueTask<bool> ValidateAsync(object? value, Cancellati
 
 		if (!Errors?.SequenceEqual(errors) ?? true)
 		{
-			Errors = errors.ToList();
+			Errors = errors;
 		}
 
 		return false;
 	}
 
 	void OnChildrenCollectionChanged(object? sender, NotifyCollectionChangedEventArgs e)
 	{
-		if (e.NewItems != null)
+		if (e.NewItems is not null)
 		{
 			foreach (var child in e.NewItems.OfType<ValidationBehavior>())
 			{
-				child.TrySetBindingContext(new Binding
-				{
-					Path = BindingContextProperty.PropertyName,
-					Source = this
-				});
+				((ICommunityToolkitBehavior<VisualElement>)child).TrySetBindingContextToAttachedViewBindingContext();
```

Comment:
if it fails do we want to log some message or throw an error?

Reply:
Agreed! I'll only use `Trace.WriteLine` when `TrySetBindingContextToAttachedViewBindingContext` fails; I won't use us when `TryRemoveBindingContext` fails since that also returns `false` when there is no `BindingContext` to remove which wouldn't necessarily be an error.

Done ✅